devops/Makefile

```diff
@@ -446,6 +449,7 @@ deploy-queue-consumers:
 		-p DATABASE_NAME_KEY=$(DATABASE_NAME_KEY) \
 		-p PORT=$(QUEUE_CONSUMERS_PORT) \
 		-p HOST_NAME=$(HOST) \
+		-p HOST_NAME=$(QUEUE_PREFIX) \
```

Comment:
Can we please remove the HOST_NAME parameter in 451, i think the idea here is to use the QUEUE_PREFIX as hostname.